tests/test_sqla_to_intermediary.py

```diff
@@ -74,6 +75,9 @@ def test_database_to_intermediary_with_schema():
     # Not in because different schema.
     assert relation not in relationships
     assert exclude_relation not in relationships
+    # Assert column names match table names, including schema
+    assert all(r.right_col in table_names for r in relationships)
+    assert all(r.left_col in table_names for r in relationships)
```

Comment:
Can you extract this part to another test and make the assertion show the "{}.{}" formatting?

Reply:
Done, separated out and checking explicitly for presence/lack of schema name in the relationship column names. Thanks for taking the time to review my updates.

Reply:
Why don't we do a set `expected_names` like and assert it's equal to the result.